client/protocols/unlock.py

```diff
@@ -150,6 +155,18 @@ def interact(self, unique_id, case_id, question_prompt, answer, choices=None, ra
 
             else:
                 correct = True
+            tg_id = -1
+            misU_count = {}
+            printed_msg = ""
+
+            if not correct:
+                misU_count, tg_id, msg = self.GuidanceUtil.guidance_msg(unique_id,input_lines,
```

Comment:
At least in this context, the method name "guidance_msg" is not clear to me. What exactly does this method do?